app/src/forms/rbac/service.js

```diff
@@ -297,6 +297,22 @@ const service = {
   getIdentityProviders: (params) => {
     return idpService.getIdentityProviders(params.active);
   },
+
```

Comment:
I think you could use `getUserForms` or `getFormUsers` instead. It's bad enough we have 2 identical methods, we don't need to add a third. We don't like logic in controllers, but I think in this case it is OK, basically just checking results and returning a boolean. So re-use one of the rbac service methods and move the results-> boolean logic to the controller. Also, boolean logic should be negative by default and find if it is positive, just ensures if something weird happens that it should not return a false positive (ie if the Array.isArray called failed, we would return true which is bad).

Reply:
I am lost here. Please can you give more details?

Reply:
So, a couple of ways to do this... since you have already updated the controller to check the array and return a boolean, you no longer need this method in the service. I think you can just have the controller call `service.getFormUsers` and remove this method (`isUserPartOfFormTeams`) from the service. I don't know why we already have 2 identical `UserFormAccess` queries in the service but we shouldn't add a third. I'll add a backlog ticket to refactor out the duplicates (`service.getFormUsers` and `service.getUserForms`).

Reply:
Fixed